src/appConfig.ts

```diff
@@ -0,0 +1,4 @@
+export default {
```

Comment:
for the future, would be much better to have here the class implementation, i.e.

```
class AppConfig {
    private config : object;
   
    constructor(config) {
        this.config = config;
    }

    get dbConnectionURI() {
        return this.config.DB_CONNECT_URL;
    }
}

export default new AppConfig(process.env)
```